en/docs/deploy-and-publish/publish-on-dev-portal/publish-aws-apis-in-the-dev-portal.md

```diff
@@ -0,0 +1,118 @@
+# Importing APIs from AWS API Gateway to the WSO2 API Manager  
+
+This feature helps publish an API a user has in the AWS API gateway, on the WSO2 developer portal. These APIs will not be deployed into the WSO2 API gateway and will not be managed by the WSO2 API gateway.
+
+## Prerequisites 
+
+1. Two WSO2 API Controller commands are needed for this task. Hence the user needs to have the APICTL downloaded and configured. The apictl is a command line interface similar to the AWS CLI. [Getting started with the API Controller](https://apim.docs.wso2.com/en/latest/learn/api-controller/getting-started-with-wso2-api-controller/)
```

Comment:
```suggestion
1. Two WSO2 API Controller (apictl) commands are needed for this task. Hence the user needs to have the apictl downloaded and configured. The apictl is a command line interface similar to the AWS CLI. [Getting started with the API Controller](https://apim.docs.wso2.com/en/latest/learn/api-controller/getting-started-with-wso2-api-controller/)
```

Reply:
IMO, the sentence "The apictl is a command line interface similar to the AWS CLI" makes apictl alien to WSO2 API-M. The WSO2 product here is apictl. So I do not think it is suitable to say that "apictl is similar to AWS CLI" since AWS CLI is not the WSO2 product.

Reply:
Agreed, we should directly compare apictl with AWS CLI, specially since this is a feature that uses the AWS CLI via the apictl, this can sound confusing. Lets just drop the sentence `The apictl is a command line interface similar to the AWS CLI`

Reply:
should i completely drop that sentence or change it to "the apictl is a command line interface" ?

Reply:
Shall we add something like this after dropping the part which you mentioned AWS CLI as @uvindra stated?

```suggestion
1. Two (2) **WSO2 API Controller (apictl)** commands are needed for this task. Hence the user needs to have the apictl downloaded and configured. (apictl is a command-line tool providing the capability to move APIs, API Products, and Applications across environments and to perform CI/CD operations. For more information, see [Getting started with the API Controller](https://apim.docs.wso2.com/en/latest/learn/api-controller/getting-started-with-wso2-api-controller/))
```

Reply:
Another tip, shall we cross-check all the links in this doc? Especially, the above link you mentioned points to the **latest** doc space which is 3.2.0. We should always have the links specified by the versions. In your case it should be **4.0.0**. Let's check if there are any occurrences like this and correct those to point to the correct doc space page (4.0.0).

Reply:
The best way to mention the links is with the base path, so that we do not need to worry about the version. 

Example:- 
[Getting Started with WSO2 API Controller (apictl)]({{base_path}}/install-and-setup/setup/api-controller/getting-started-with-wso2-api-controller)